platform/darwin/src/MGLAccountManager.h

```diff
@@ -1,11 +1,12 @@
-#import <Foundation/Foundation.h>
```

Comment:
I’m not sure why, but we don’t seem to be affected by realm/jazzy#609 even with these changes. Running `make xdocument` turns up some broken Swift declarations, but that happens on master too. (@incanus may have a stronger opinion than I about explicitly importing frameworks in headers. My concern was limited to jazzy output.)